projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/featured-image/index.tsx

```diff
@@ -3,22 +3,27 @@
  */
 import { useImageGenerator } from '@automattic/jetpack-ai-client';
 import { Button, Spinner } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
 import { useCallback, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
 import './style.scss';
 import usePostContent from '../../hooks/use-post-content';
+import useSaveToMediaLibrary from '../../hooks/use-save-to-media-library';
 import AiAssistantModal from '../modal';
 
 const FEATURED_IMAGE_FEATURE_NAME = 'featured-post-image';
 
 export default function FeaturedImage() {
+	const { editPost } = useDispatch( editorStore );
```

Comment:
We need to test this on the P2 editor, the `editorStore` const may not work there, just the literal string for the store (`core/editor`, I guess).